src/yunohost/data_migrations/0011_setup_group_permission.py

```diff
@@ -85,14 +82,18 @@ def migrate_app_permission(self, app=None):
             domain = app_setting(app, 'domain')
 
             urls = [domain + path] if domain and path else None
-            permission_add(app, permission='main', urls=urls, default_allow=True, sync_perm=False)
+            permission_create(app+".main", urls=urls, sync_perm=False)
             if permission:
                 allowed_group = permission.split(',')
-                user_permission_add([app], permission='main', group=allowed_group, sync_perm=False)
+                user_permission_update(app+".main", remove="all_users", add=allowed_group, sync_perm=False)
             app_setting(app, 'allowed_users', delete=True)
 
-
     def run(self):
+
+        # FIXME : what do we really want to do here ...
+        # Imho we should just force-regen the conf in all case, and maybe
+        # just display a warning if we detect that the conf was manually modified
+
```

Comment:
Well, as I understand you want to force the regen-conf here ?

Reply:
Yes, what are your thoughts on this ?

Reply:
As I remember I did this and I changed because we said that we can't drop the custom config of the user in case of a custom config.

Reply:
Anyway I'm for the idea to add this. We might avoid many issue if we force the regen-conf here I think.